src/app/static/about/about.component.html

```diff
@@ -86,152 +86,152 @@ <h2>{{ 'anms.about.contributors' | translate }}</h2>
     <div class="contributors-wrapper">
 
       <!-- ALL-CONTRIBUTORS-LIST:START -->
-  <a class="contributor" href="https://medium.com/@tomastrajan" target="_blank">
-    <img src="https://avatars0.githubusercontent.com/u/3764868?v=4">
+  <a class="contributor" href="https://medium.com/@tomastrajan" target="_blank" attr.aria-label="{{'anms.a11y.about.contributors-link' | translate}}">
+    <img src="https://avatars0.githubusercontent.com/u/3764868?v=4" alt="{{'anms.a11y.about.contributors-avatar' | translate}}">
```

Comment:
I think we can use `aria hidden` for these images.

Reply:
i'll add it to the loop : 
```
<a *ngFor="let c of contributors" class="contributor" [href]="c.profile" target="_blank" [attr.aria-label]="'anms.a11y.about.contributors-link' | translate:{name: c.name}">
        <img [src]="c.avatar_url">
        <span>{{c.name}}</span>
</a>
```

Reply:
Would it be worth using trackBy here and throughout the rest of the app to promote good practice? I know this list is unlikely to change dynamically but others such as todo's might and if it did the DOM for the entire list would be re-rendered instead of just the changes